ale_linters/pyrex/cython.vim

```diff
@@ -1,10 +1,27 @@
-" Author: w0rp <devw0rp@gmail.com>
+" Author: w0rp <devw0rp@gmail.com>,
+" Nicolas Pauss <https://github.com/nicopauss>
 " Description: cython syntax checking for cython files.
 
+call ale#Set('pyrex_cython_executable', 'cython')
+call ale#Set('pyrex_cython_options', '--warning-extra --warning-errors')
```

Comment:
Could you remove `--warning-errors` here, so we can preserve the current behavior? If users want warnings to be errors, they can change the option themselves. I don't want to change the behavior without a very compelling reason, and whether warnings should be errors is subjective.

Reply:
Done.